packages/drizzle-duckdb-wasm/package.json

```
@@ -16,48 +16,49 @@
   },
   "exports": {
     ".": {
-      "types": "./dist/index.d.ts",
+      "types": "./dist/index.d.mts",
       "import": "./dist/index.mjs",
-      "node": "./dist/index.cjs"
+      "node": "./dist/index.mjs"
     },
     "./bundles/default-browser": {
-      "types": "./dist/bundles/default-browser.d.ts",
+      "types": "./dist/bundles/default-browser.d.mts",
       "import": "./dist/bundles/default-browser.mjs"
     },
     "./bundles/default-node": {
-      "types": "./dist/bundles/default-node.d.ts",
+      "types": "./dist/bundles/default-node.d.mts",
       "import": "./dist/bundles/default-node.mjs",
-      "node": "./dist/bundles/default-node.cjs"
+      "node": "./dist/bundles/default-node.mjs"
     },
     "./bundles/import-url-browser": {
-      "types": "./dist/bundles/import-url-browser.d.ts",
+      "types": "./dist/bundles/import-url-browser.d.mts",
       "import": "./dist/bundles/import-url-browser.mjs"
     },
     "./bundles/import-url-node": {
-      "types": "./dist/bundles/import-url-node.d.ts",
+      "types": "./dist/bundles/import-url-node.d.mts",
       "import": "./dist/bundles/import-url-node.mjs",
-      "node": "./dist/bundles/import-url-node.cjs"
+      "node": "./dist/bundles/import-url-node.mjs"
     }
   },
-  "main": "./dist/index.cjs",
+  "main": "./dist/index.mjs",
   "module": "./dist/index.mjs",
-  "types": "./dist/index.d.ts",
+  "types": "./dist/index.d.mts",
   "files": [
     "README.md",
     "dist",
     "package.json"
   ],
   "scripts": {
-    "dev": "pnpm run stub",
-    "stub": "unbuild",
-    "build": "unbuild && pnpm run play:build",
+    "dev": "tsdown",
+    "stub": "tsdown",
+    "build": "tsdown",
     "play:dev": "vite",
     "play:build": "vite build",
     "play:preview": "vite preview",
```

Trying to fix Netlify failures:

```suggestion
    "dev:play": "vite",
    "build:play": "vite build",
    "preview:play": "vite preview",
```